### ISSUE TITLE
IEX Cloud: Wrong correlation between minute bars and historic prices #2

### ISSUE THREAD

**Issue by stellarator** [iexg/IEX-API#789]
Colleagues, I really don't like to produce duplicates. But it seems that month has passed since the creation of the original [issue](https://github.com/iexg/IEX-API/issues/684), but I have not received a clear answer.

EACH day introduces errors in 1d data. To confirm my words - this is data for the only one (pretty known) symbol and the last week:
```
                                   |       minute bars aggregate       ||        historic (EOD) data
 timestamp  |        iex_id        |  open  |  high  |  low   |  close ||  open  |  high  |  low   |  close

 2019-03-11 | IEX_4D48333344362D52 | 175.49 | 179.13 | 172.91 | 178.98 || 175.49 | 179.12 | 175.35 | 178.90
 2019-03-12 | IEX_4D48333344362D52 | 179.93 | 182.67 | 178.90 | 180.94 || 180.00 | 182.67 | 179.37 | 180.91
 2019-03-13 | IEX_4D48333344362D52 | 182.29 | 183.30 | 172.91 | 181.72 || 182.25 | 183.30 | 180.92 | 181.71
 2019-03-14 | IEX_4D48333344362D52 | 183.91 | 184.10 | 178.73 | 183.73 || 183.90 | 184.10 | 182.56 | 183.73
 2019-03-15 | IEX_4D48333344362D52 | 184.78 | 187.33 | 183.73 | 186.21 || 184.85 | 187.33 | 183.74 | 186.12
```
Please, pay special attention to the `low` columns (at least).

The questions are:
- When we start to receive so called `high quality` data?
- How will costs be compensated for already downloaded `bad/garbage` data?
- Will it be possible to reload correct data for previous periods by paying the regular `1d` endpoint price for them? Actually - this is not the question but assumption (because I started fetching data from '2019-02-15', for example).

Thanks in advance.

CC: @joshuablackburn-iex, @evancarter-iex

**Comment by stellarator** [comment]
I can paraphrase. Letters with roadmap, changes in site's personal area, etc - all of this are cool things. But they costs nothing until we can work with data that we can trust. And here I mean trust in global term - to you, to the platform, to the promises and actions from your side.

Skip a while... Whistles and fakes can be added later. Make the data correct and accessible to people. I mean it was the original intent isn't it?

**Comment by joshuablackburn-iex** [comment]
I appreciate your very direct feedback and your patience. We recognize there may be inconsistencies between different data sets, and we are working to resolve them. By no means do we intend to make excuses, but I do want to highlight what we have identified so far:

- Some data alignment issues have come up during the migration of data providers while trying to keep as much consistency with the old API to ease existing user workloads. In some cases it is better if we deprecate old endpoints and start with something that makes more sense for the current data set. This has created issues where it is not clear what type of data we are returning.

- That data you present above comes from a difference between how we (IEX) collect intraday trades versus how our new partner collects trades. We had aligned with our old provider before, and our intraday data included non last sale eligible trades. Those volume only eligible trades are creating differences in the high and low values. As of today our intraday prices are reflecting last sale eligible only trades. It is worth noting that intraday data can differ from historical data once everything has been processed. This can be the result of late reported trades or trade busts. Similar to my point above, we need to do a better job of breaking out this endpoint. Intraday minute bar data should not be commingled with historical data, and misrepresents the full accuracy of intraday data. We are working on historical minute bar data that will be fully cleaned for all cases.

- We appreciate feedback from all our users and recognize that everyone has different needs. We are also working with many 3rd parties which have their own timelines. We deploy our resources to make the most impact for all users, and data quality is one of our highest priorities.
